readthedocs/doc_builder/backends/sphinx.py

```diff
@@ -121,16 +121,22 @@ def append_conf(self, **__):
         """Modify given ``conf.py`` file from a whitelisted user's project."""
         try:
             self.version.get_conf_py_path()
-        except ProjectImportError:
+        except ProjectConfigurationError:
             master_doc = self.create_index(extension='rst')
             self._write_config(master_doc=master_doc)
 
         try:
             outfile_path = self.project.conf_file(self.version.slug)
             outfile = codecs.open(outfile_path, encoding='utf-8', mode='a')
-        except (ProjectImportError, IOError):
+        except (ProjectConfigurationError, IOError):
             trace = sys.exc_info()[2]
-            six.reraise(ProjectImportError('Conf file not found'), None, trace)
+            six.reraise(
```

Comment:
Why do we need this `six.reraise`? It's not the same than just `raise ProjectImportError(...)`?

Reply:
I'm not sure either, the docs aren't clear to me:
https://pythonhosted.org/six/#six.reraise

I'll leave it, assuming someone with more knowledge of py2/3 compat did this :)